docs/content/latest/api/ysql/commands/ddl_create_index.md

```diff
@@ -130,3 +138,14 @@ yugabyte=# \d products_name_code;
  code   | text | no   | code
 lsm, for table "public.products"
 ```
+
+
+### Partial indexes
+
+For example, an application maintaining a `shipments` table may have a column for shipment status. 
+If the application mostly accesses the in-flight shipments, then it can use a partial index to exclude rows whose shipment status is `delivered`.
```

Comment:
```suggestion
Consider an application maintaining shipments information. It has a `shipments` table with a column for `delivery_status`. If the application needs to access in-flight shipments frequently, then it can use a partial index to exclude rows whose shipment status is `delivered`.
```